src/ConditionalStatementByPawan/ifTrueCondition.java

```diff
@@ -0,0 +1,12 @@
+package ConditionalStatementByPawan;
+
+public class ifTrueCondition {
+
+    public static void main(String[]args){
+        if(true){
```

Comment:
what if we make condition false

Reply:
Addressd the comment and make condition as false